susemanager-utils/testing/docker/head/uyuni-head-gatherer/add_packages.sh

```diff
@@ -7,7 +7,7 @@ zypper --non-interactive --gpg-auto-import-keys ref
 # this package has a file conflict with python-base
 # and this package is not on SLES12 SP1
 # a lock prevent the installation
-zypper addlock python-strict-tls-check
+#zypper addlock python-strict-tls-check
```

Comment:
Shouldn't we just remove this line?

Reply:
Yes, but I wanted to try if we can build without it.